catalog/harvest/catalog-harvest-common/src/main/java/org/codice/ddf/catalog/harvest/common/FileSystemPersistenceProvider.java

```diff
@@ -117,7 +126,9 @@ public void storeAll(Map<String, Object> keyValueMap) {
   public void delete(String key) {
     File file = new File(getMapStorePath() + key + PERSISTED_FILE_SUFFIX);
     if (file.exists()) {
-      if (!file.delete()) {
+      try {
+        Files.delete(file.toPath());
+      } catch (IOException e) {
         LOGGER.debug("File was unable to be deleted: {}", file.getAbsolutePath());
```

Comment:
❓Is this going to cause a problem upon restart? Should we log something higher than `debug`?